packages/collaboration-extension/src/collaboration.ts

```diff
@@ -189,3 +191,63 @@ export const userEditorCursors: JupyterFrontEndPlugin<void> = {
     });
   }
 };
+
+export const notebookCellExecutor: JupyterFrontEndPlugin<INotebookCellExecutor> =
+  {
+    id: '@jupyter/collaboration-extension:notebook-cell-executor',
+    description:
+      'Add notebook cell executor that uses REST API instead of kernel protocol over WebSocket.',
+    autoStart: true,
+    provides: INotebookCellExecutor,
+    activate: (app: JupyterFrontEnd): INotebookCellExecutor => {
+      if (PageConfig.getOption('serverSideExecution') === 'true') {
```

Comment:
Looking at the diff for this PR, this `serverSideExecution` page config option does not seem to be set?

Reply:
My understanding is that setting it [here](https://github.com/jupyterlab/jupyter-collaboration/blob/a0f46107375c8ba0fb19590fae13c1ce6c78c0b5/jupyter_collaboration/app.py#L60) magically makes it appear in the `PageConfig` as `serverSideExecution`.
I checked that it can be configured by Jupyverse's CLI:
```bash
jupyverse --set kernels.require_yjs=true --set jupyterlab.server_side_execution=true
```

Reply:
> My understanding is that setting it [here](https://github.com/jupyterlab/jupyter-collaboration/blob/a0f46107375c8ba0fb19590fae13c1ce6c78c0b5/jupyter_collaboration/app.py#L60) magically makes it appear in the `PageConfig` as `serverSideExecution`.


Isn't because it is set here in `jupyverse`?

https://github.com/jupyter-server/jupyverse/blob/085705a82c77718be428ddde67ea1e52af4b1e3a/plugins/jupyterlab/fps_jupyterlab/routes.py#L127

Reply:
Oh indeed, I forgot about this one. So no magic :smile: 
I'll make the change in the settings, thanks.